### DISCUSSION TITLE
Remove users after they lose the role

### DISCUSSION THREAD

**Discussion by mosteanuv** [Yoruio/Membarr#57] [General]
Hey guys,

Love the app but I have a question.

Is it possible to make Membarr take away the Plex access after a user loses the role setup to give access?

**Comment by coolcat97** [comment]
agree

**Comment by Yoruio** [comment]
This should already happen... Check your logs if this is not happening to see if there are any errors, and send them to me!

**Reply by coolcat97** [reply]
Where in the logs would this be seen?

**Reply by coolcat97** [reply]
and how do i test?
